Add origami-version GitHub Action

JakeChampion opened the pull request.

origamiserviceuser commented:
o-comment-utilities bundle size difference from 2.4.0 to 9cbc1d7520cfade96d87a56de2223183cbb28f1c
No significant bundle size differences found.

JakeChampion committed:
Add origami-version GitHub Action

When this is merged, we can have automated releases by using the release-type labels "major"/"minor"/"patch" on a pull-request before merging the pull-request.

origamiserviceuser commented:
o-comment-utilities bundle size difference from 2.4.0 to 764f667e28e4bd70275a83c9a27be2dca6b3b7dd
No significant bundle size differences found.
